### PR TITLE
Include videos in courses page, other minor changes

### PR DESCRIPTION
**Pull request by morsssss**
Include area to showcase our new coursework videos!
Change name and order of courses landing page in sidebar

### CODE REVIEW

File: pages/content/amp-dev/documentation/courses/index.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-$title: Learn AMP with free courses
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/courses/index.html]: I actually liked that this showed what kind of resource the user can expect over other potential SERP items 🙂Were there specific reasons going for just "Learn AMP"?

**Comment by morsssss** [comment]
@matthiasrohmer , just added here the CSS tweak from #1913 .

File: pages/content/amp-dev/documentation/courses/index.html (hunk 1 above)
Comment (morsssss) [reply pages/content/amp-dev/documentation/courses/index.html]: Yes - I thought the previous title looked too long to me in the sidebar - too much like an ad. I hadn't realized that this is what also appears in the SERP. Of course, it is. Don't suppose there's a way to get one thing in the sidebar and another in the `<title>` tag?

**Comment by matthiasrohmer** [comment]
I am more than okay with my changes being submitted!

**Pull request merged**